data/brands/shop/car_parts.json

```
@@ -12,6 +12,16 @@
     }
   },
   "items": [
+        {
+      "displayName": "Tiger Wheel & Tyre",
+      "locationSet": {"include": ["za"]},
+      "tags": {
+        "brand": "Tiger Wheel & Tyre",
+        "brand:wikidata": "Q120762656",
+        "name": "Tiger Wheel & Tyre",
+        "shop": "car_parts"
```

So this already exists under `shop=tyres`, but I think you might be right. Can you remove it from that file please.

Also the website suggests they do the following at all of the locations:

```suggestion
        "shop": "car_parts",
        "service:vehicle:tyres": "yes",
        "service:vehicle:car_parts": "yes",
        "service:vehicle:batteries": "yes",
        "service:vehicle:wheels": "yes"
```

---

I'll remove it from tyres.  Thanks.

---

I'll remove it from tyres.  Thanks.